ubuntu.22.04/Dockerfile

```diff
@@ -120,12 +119,11 @@ RUN wget --quiet https://releases.hashicorp.com/terraform/${Terraform_Version}/t
     mv terraform /usr/local/bin && \
     rm terraform_${Terraform_Version}_linux_amd64.zip
 
-
 # Install Google Cloud CLI
 # https://cloud.google.com/sdk/docs/downloads-apt-get
 RUN curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | gpg --dearmor -o /usr/share/keyrings/cloud.google.gpg && \
     echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] https://packages.cloud.google.com/apt cloud-sdk main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list && \
-    apt-get update && apt-get install -y google-cloud-sdk=${Google_Cloud_Cli_Version} google-cloud-cli=${Google_Cloud_Cli_Version} google-cloud-sdk-gke-gcloud-auth-plugin=${Google_Cloud_Gke_Cloud_Auth_Plugin_Version} && \
+    apt-get update && apt-get install -y google-cloud-cli=${Google_Cloud_Cli_Version} google-cloud-cli-gke-gcloud-auth-plugin=${Google_Cloud_Gke_Cloud_Auth_Plugin_Version} && \

```

Comment:
@Justin-Walsh did the package names change? is `-sdk-` gone now?

Reply:
Yeah, SDK was discontinued and replaced with the CLI packages. The last version of the SDK in the repo was 476 or so.